src/plugins/poster/poster.js

```diff
@@ -42,12 +42,8 @@ export default class PosterPlugin extends UIContainerPlugin {
     this.listenTo(this.container, Events.CONTAINER_STOP, this.onStop)
     this.listenTo(this.container, Events.CONTAINER_PLAY, this.onPlay)
     this.listenTo(this.container, Events.CONTAINER_ENDED, this.onStop)
-    if (this.container.buffering) {
-      process.nextTick(() => this.onBuffering())
```

Comment:
I didn't, but this was something I put in a couple of commits ago, and the same should be being achieved now with `update()` being called at the end of render, and the `.buffering` check in render anyway